cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/tests/tests.rs

```diff
@@ -271,22 +270,6 @@ fn relayed_incoming_message_works() {
 	)
 }
 
-#[test]
-pub fn complex_relay_extrinsic_works() {
-	from_parachain::complex_relay_extrinsic_works::<RuntimeTestsAdapter>(
```

Comment:
so the complex relayer won't work with older `submit_finality_proof` even if it pays ?

Reply:
It should work, but his transactions won't be refunded + we do not check for obsolete stuff inside batches anymore. This tests checks that after complex transaction, relayer receives a refund, so it fails. There is a test somewhere nearby for a standalone transactions

Reply:
@bkontur I've added tests in this commit: https://github.com/paritytech/polkadot-sdk/pull/4102/commits/cde347f7eb86eee4bab73638a4fc5133226394f0 PTAL. Thank you!

Reply:
> @bkontur I've added tests in this commit: [cde347f](https://github.com/paritytech/polkadot-sdk/commit/cde347f7eb86eee4bab73638a4fc5133226394f0) PTAL. Thank you!

ok, checked, cool, thank you :)